test-cases/scale/longevity-elasticity-many-small-tables.yaml

```diff
@@ -0,0 +1,35 @@
+# This is a test case for having 90% disk utilization with a lot of small tables.
+# The data is split equally among 500 tables.
+# The dataset size is aligned with 'i4i.xlarge' and RF = 3 = number-of-DB-nodes.
+# It uses a c-s user-profile template for all 500 tables.
+# It runs 4 batches of 125 tables each.
+# On each batch cycle, 125 tables are created, then a load is generated for all of these tables.
+# When all the 125 stress writes/reads are done, it continue with the next batch until stress to all 500 tables is completed (after 4 cycles).
+# Each one of the 500 tables has both write and read load.
+
+test_duration: 600
+
+cs_user_profiles:
+    - data_dir/templated-elasticity-tables.yaml
+
+pre_create_schema: true
+user_profile_table_count: 500
+batch_size: 125
```

Comment:
why with 500 tables and 125 per batch test ended with 8 batches?

Reply:
@soyacz , please see the answer to this question in https://github.com/scylladb/scylla-cluster-tests/pull/9785#issuecomment-2609786024